Makefile

```diff
@@ -58,7 +63,12 @@ CALYPSO_ENV ?= $(NODE_ENV)
 
 export NODE_ENV := $(NODE_ENV)
 export CALYPSO_ENV := $(CALYPSO_ENV)
-export NODE_PATH := server:client:.
+export NODE_PATH := server$(SEPARATOR)client$(SEPARATOR).
+ifeq ($(OS),Windows_NT)
+export NPM_GLOBAL_ROOT := $(shell $(NPM) -g root)
```

Comment:
It's just a different path - on Windows it's just in what `npm -g root` returns, while on *nixes you a weird "duplicated" path, like `/usr/lib64/node_modules/npm/node_modules/` (`/usr/lib64/node_modules` is what `npm root -g` returns, and then you have to go again into `node_modules`? :confounded:).
Not sure how to document this - it's just a different path on one platform than the other.